tests/unit/helpers.js

```diff
@@ -241,6 +241,12 @@ export function createFakeEvent(extraProps = {}) {
   };
 }
 
+export function createFakeChangeEvent(value = '') {
```

Comment:
Another option is to move `createFakeChangeEvent` directly into `TestSearchForm.js` since that's the only place it's used.